src/peerd/runtime.rs

```diff
@@ -334,62 +337,69 @@ impl esb::Handler<ServiceBus> for Runtime {
 
 impl Runtime {
     /// send messages over the bridge
-    fn handle_rpc_msg(
+    fn handle_msg(
         &mut self,
         endpoints: &mut Endpoints,
         _source: ServiceId,
-        request: Request,
+        message: Msg,
     ) -> Result<(), Error> {
-        match request.clone() {
-            Request::Protocol(message) => {
-                // 1. Check permissions
-                // 2. Forward to the remote peer
-                debug!("Message type: {}", message.get_type());
-                debug!(
-                    "Forwarding peer message to the remote peer, request: {}",
-                    &request.get_type()
-                );
-                self.messages_sent += 1;
-                while let Err(err) = self.peer_sender.send_message(message.clone()) {
-                    debug!("Error sending to remote peer in peerd runtime: {}", err);
-                    // If this is the listener-forked peerd, i.e. the maker's peerd, terminate it.
-                    if self.forked_from_listener {
-                        endpoints.send_to(
-                            ServiceBus::Ctl,
-                            self.identity(),
-                            ServiceId::Farcasterd,
-                            Request::PeerdTerminated,
-                        )?;
-                        warn!("Exiting peerd");
-                        std::process::exit(0);
-                    }
-
-                    while let Err(err) = self.reconnect_peer() {
-                        warn!("error during reconnection attempt: {}", err);
-                    }
-                }
+        // 1. Check permissions
```

Comment:
we're not checking permissions here (yet)

Reply:
Not a new line, was originally in code refactored. I’ll add it as a todo

Reply:
Done in [f87e4de](https://github.com/farcaster-project/farcaster-node/pull/707/commits/f87e4de068e9904b24cbb6fbe5ea4776496ddd2d)